.github/workflows/audit.yml

```diff
@@ -0,0 +1,20 @@
+name: Cargo Audit
```

Comment:
We might want to unify this job with the existing circle workflow instead?

Reply:
Could do. Is that preferred?